spec/unit/puppet/provider/network_vlan/cisco_nexus_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'spec_helper'
+require 'cisco_node_utils'
+
+ensure_module_defined('Puppet::Provider::NetworkVlan')
+require 'puppet/provider/network_vlan/cisco_nexus'
+
+RSpec.describe Puppet::Provider::NetworkVlan::CiscoNexus do
+  let(:provider) { described_class.new }
+  let(:context) { instance_double('Puppet::ResourceApi::BaseContext', 'context') }
+  let(:device) { instance_double('Puppet::Util::NetworkDevice::Nexus::Device', 'device') }
+  let(:networkvlan) { instance_double('Cisco::NetworkVlan', 'networkvlan') }
+
+  let(:vlan) { instance_double('Cisco::Vlan', 'vlan') }
+  let(:vlan2) { instance_double('Cisco::Vlan', 'vlan2') }
+  let(:vlans) do
+    {
+      '1' => vlan,
+      '2' => vlan2,
+    }
+  end
+  let(:should_values) do
+    {
+      id: '42',
+      vlan_name: 'vlan_42',
+      ensure: 'present',
+      shutdown: true,
+    }
+  end
+  let(:changes) do
```

Comment:
From what I can see this is not used - can be removed?

Reply:
Removed. I think this was from an earlier iteration of tests before I refactored them as simple provider is the main set method.